certbot/plugins/util.py

```diff
@@ -24,6 +35,50 @@ def already_listening(port, renewer=False):
     :returns: True or False.
 
     """
+
+    if USE_PSUTIL:
+        return already_listening_psutil(port, renewer=renewer)
```

Comment:
Err on second thought, we want `already_listening_socket/psutil` to always behave the same, so changing their behavior should happen in one place: `already_listening`. So nvm lol.